pkg/minikube/machine/cache_images.go

```diff
@@ -106,24 +108,26 @@ func LoadImages(cc *config.MachineConfig, runner command.Runner, images []string
 		return errors.Wrap(err, "runtime")
 	}
 
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		glog.Infof("error retrieving oci daemon client %v", err)
+		return err
```

Comment:
Does this add a dependency on a local Docker daemon?

Reply:
it uses the same lib that go-container lib uses. it doesnt need user to install docker but uses the docker lib.

Reply:
it would worth that I test personally with other than docker, (crtictl) and see if the docker lib would still retrieve it. I will test that update the comment

Reply:
just did a local test and it will work without a docker cli installed.